mscore/inspector/inspector.cpp

```diff
@@ -678,7 +678,8 @@ InspectorKeySig::InspectorKeySig(QWidget* parent)
       const std::vector<InspectorItem> iiList = {
             { Pid::LEADING_SPACE,  1, s.leadingSpace,  s.resetLeadingSpace  },
             { Pid::SHOW_COURTESY,  0, k.showCourtesy,  k.resetShowCourtesy  },
-//          { Pid::SHOW_NATURALS,  0, k.showNaturals,  k.resetShowNaturals  }
+          //{ Pid::SHOW_NATURALS,  0, k.showNaturals,  k.resetShowNaturals  },
+            { Pid::KEYSIG_MODETYPE,0, k.modeType,      k.resetModeType      }
```

Comment:
This is not enough to make the combo works. You need to define user data in the combo to match the value of the Keymode enum. If not, MuseScore uses enum value as index `void InspectorBase::setValue(const InspectorItem& ii, QVariant val)`

Reply:
OK, this might be it then.

Reply:
Now at least that menu forsn't start empty, but with "Unknown", still the setting doesn't stick though